app/src/ui/business-card/BusinessCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import clsx from "clsx";
+import styles from "./BusinessCard.module.scss";
+
+import { Card } from "ui/card/Card";
+import { Grid } from "ui/grid/Grid";
+import { Container, Visible } from "react-grid-system";
+import { Typography } from "ui/typography/Typography";
+import { BusinessCardProps } from "./BusinessCard.types";
+import { useRouter } from "next/router";
+
+export const BusinessCard: React.FC<BusinessCardProps> = ({ pageLink, content, className }) => {
+  //@TODO Implement i18n
+
+  const router = useRouter();
+
+  const truncateRaisedAmount = (raised: number) => {
+    if (!raised) return "0";
+
+    const truncated = Intl.NumberFormat("en-US", {
+      notation: "compact",
+    }).format(raised);
+
+    return truncated;
+  };
+
+  const redirectToBusinessPage = (_event: React.MouseEvent<HTMLElement>) => {
+    router.push(pageLink);
+  };
+
+  return (
+    <div className={clsx(styles["business-card"], className)} onClick={redirectToBusinessPage}>
+      <Container>
+        <Grid.Row>
+          <Grid.Col sm={6} lg={4} xl={6}>
```

Comment:
@Ktoxcon yes, this is the way. The Card will take the column space, and we don't mess with static `width` s